### PR TITLE
fix(mapping): generate mapping from dictionary, instead of RDD

### PR DESCRIPTION
**Pull request by m0nhawk**

### CODE REVIEW

File: tube/spark/translator.py [hunk 1]
```diff
@@ -161,8 +163,44 @@ def get_direct_children(self, root_df):
             root_df = root_df.leftOuterJoin(child_df).mapValues(lambda x: merge_and_fill_empty_fields(x, n.fields))
         return root_df
 
+    def get_types(self):
```
Comment (thanh-nguyen-dang) [inline tube/spark/translator.py]: I think we should move this part to the parser and save it as a property of parser.

File: tube/spark/translator.py [hunk 2]
```diff
@@ -161,8 +163,44 @@ def get_direct_children(self, root_df):
...
+            if k == "_flatten_props":
+                for i in v:
+                    a = get_properties_types(models, get_attribute_from_path(models, root, i['path']))
+                    for j in i['_props']:
+                        types[j] = a[j]
+
+            if k == "_props":
+                types.update({w: get_properties_types(models, root)[w] for w in v})
+
+        for k, v in types.items():
```
Comment (thanh-nguyen-dang) [inline tube/spark/translator.py]: can we move this part to another function called `choosing_type` or something like that

File: tube/spark/translator.py (hunk 1 above)
Comment (m0nhawk) [reply tube/spark/translator.py]: Will do.

File: tube/spark/translator.py (hunk 2 above)
Comment (m0nhawk) [reply tube/spark/translator.py]: Will do.

**Pull request merged**